From 80db50a5dd0e85daa11bd246348f5a5d902d9637 Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Thu, 12 Mar 2015 16:38:39 +0800
Subject: [PATCH] MA-6387 Update android kernel config to align with
 imx_3.14.y kernel update

Update android config due to imx_v7_defconfig update in latest merge
Remove duplicated config for RTL with "make savedefconfig"

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 arch/arm/configs/imx_v7_android_defconfig | 8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/arch/arm/configs/imx_v7_android_defconfig b/arch/arm/configs/imx_v7_android_defconfig
index f75e737..2107e8d 100644
--- a/arch/arm/configs/imx_v7_android_defconfig
+++ b/arch/arm/configs/imx_v7_android_defconfig
@@ -187,9 +187,6 @@ CONFIG_RFKILL_INPUT=y
 CONFIG_DEVTMPFS=y
 CONFIG_DEVTMPFS_MOUNT=y
 # CONFIG_STANDALONE is not set
-CONFIG_RTLWIFI=m
-CONFIG_RTLWIFI_DEBUG=y
-CONFIG_RTL8821AS=m
 CONFIG_DMA_CMA=y
 CONFIG_CMA_SIZE_MBYTES=320
 CONFIG_IMX_WEIM=y
@@ -266,6 +263,7 @@ CONFIG_ATH_CARDS=m
 CONFIG_ATH6KL=m
 CONFIG_ATH6KL_SDIO=m
 CONFIG_BRCMFMAC=m
+CONFIG_RTL8821AS=m
 # CONFIG_INPUT_MOUSEDEV_PSAUX is not set
 CONFIG_INPUT_EVDEV=y
 CONFIG_INPUT_EVBUG=y
@@ -381,6 +379,7 @@ CONFIG_FB_MXC_MIPI_DSI=y
 CONFIG_FB_MXC_TRULY_WVGA_SYNC_PANEL=y
 CONFIG_FB_MXC_HDMI=y
 CONFIG_FB_MXC_EINK_PANEL=y
+CONFIG_FB_MXC_EINK_V2_PANEL=y
 CONFIG_FB_MXS_SII902X=y
 CONFIG_FB_MXC_DCIC=y
 CONFIG_HANNSTAR_CABC=y
@@ -395,6 +394,7 @@ CONFIG_SND_IMX_SOC=y
 CONFIG_SND_SOC_EUKREA_TLV320=y
 CONFIG_SND_SOC_IMX_CS42888=y
 CONFIG_SND_SOC_IMX_WM8958=y
+CONFIG_SND_SOC_IMX_WM8960=y
 CONFIG_SND_SOC_IMX_WM8962=y
 CONFIG_SND_SOC_IMX_SGTL5000=y
 CONFIG_SND_SOC_IMX_MQS=y
@@ -462,6 +462,7 @@ CONFIG_HID_ZEROPLUS=y
 CONFIG_HID_ZYDACRON=y
 CONFIG_USB_HIDDEV=y
 CONFIG_USB_ANNOUNCE_NEW_DEVICES=y
+CONFIG_USB_OTG_FSM=y
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_EHCI_MXC=y
 CONFIG_USB_ACM=y
@@ -503,6 +504,7 @@ CONFIG_RTC_DRV_MXC=y
 CONFIG_RTC_DRV_SNVS=y
 CONFIG_DMADEVICES=y
 CONFIG_MXC_PXP_V2=y
+CONFIG_MXC_PXP_V3=y
 CONFIG_IMX_SDMA=y
 CONFIG_MXS_DMA=y
 CONFIG_STAGING=y
-- 
1.8.0

